src/components/state-history-chart-timeline.js

```diff
@@ -174,7 +174,7 @@ class StateHistoryChartTimeline extends LocalizeMixin(PolymerElement) {
     };
 
     const formatTooltipBeforeBody = (item, data) => {
-      if (!item[0]) {
+      if (!this.hass.userData?.showAdvanced || !item[0]) {
```

Comment:
This is a javascript file, not typescript. So you can't use optional chaining.

Reply:
Sorry, that was a copy paste error. I have applied your suggestion.